datagrid/datagrid65-basic.json

```diff
@@ -32,6 +32,30 @@
             "name": "IMAGE_STREAM_NAMESPACE",
             "value": "openshift",
             "required": true
+        },
+        {
+            "description": "The name of the secret containing the keystore file",
+            "name": "DATAGRID_JGROUPS_ENCRYPT_SECRET",
```

Comment:
We should probably s/DATAGRID/JDG/ here to be consistent with the security env vars.